lib/routes/account.js

```diff
@@ -1420,6 +1420,77 @@ module.exports = function (
       }
     },
     {
+      method: 'GET',
+      path: '/account/sessions',
+      config: {
+        auth: {
+          strategy: 'sessionToken'
+        },
+        response: {
+          schema: isA.array().items(isA.object({
+            id: isA.string().regex(HEX_STRING).required(),
+            lastAccessTime: isA.number().min(0).required().allow(null),
+            lastAccessTimeFormatted: isA.string().optional().allow(''),
+            userAgent: isA.string().max(255).required().allow(''),
```

Comment:
I don't think `userAgent` would ever be `''` because of it is set [here](https://github.com/mozilla/fxa-auth-server/blob/45d8a7ade73acef9ed73b9f5ecc3bb2da8919da0/lib/routes/account.js#L1464). I guess it could if you wanted to `trim` the string before sending.